manticore/ethereum/abi.py

```diff
@@ -3,7 +3,22 @@
 import uuid
 
 import re
-import sha3
+
+try:
+    # Older Python versions use pysha3
+    import sha3
+except ImportError:
+    # Python 3.11+ doesn't have pysha3, use pycryptodome as sha3
+    from Crypto.Hash import keccak
+
+    class sha3:
+        @staticmethod
+        def keccak_256(data=b""):
+            k = keccak.new(digest_bits=256)
+            if data:
+                k.update(data)
+            return k
+
```

Comment:
can we replace all users of sha3 with pycryptodome directly on all python versions instead of this try-except-wrap solution? see e.g. https://github.com/elopez/manticore/commit/cddb7c038c83e9d65e456bbab8290062a7bffeaa